alwayson/models.py

```diff
@@ -8,17 +8,34 @@
 SECRET = 'always-on-secret-key'
 
 
-# TODO I Should have an AlwaysOn Request/Response Exchange
-# object where the encode and decode stuff can be abstracted
+class JWTEncodeable():
+  def encode(self) -> str:
+    # TODO I may just be able to use this as a self.__dict__ or something and it works for everything
+    # TODO I may also want this to take in a SECRET and org/tenant...actually that will be encoded
+    # by the SECRET since that will be specific to each org/tenant.
+    pass
+
+
 class AlwaysOnResponse(BaseModel):
   status_code: int # TODO could probably further restruct this to valid HTTP status codes.
   body: Union[Dict[str, Any], List[Dict]]
 
+  # TODO This is 100% a misplaced responsibility since it's decoding a list of AlwaysOnResponses
```

Comment:
Could a @staticmethod make sense in that case? 🤷

Reply:
I haven't fully decided but I think it would be better to have a utility function that handles encoding/decoding. My other thought was writing a class that represents multiple AlwaysOnResponses and sticking this class method there.